index.bs

```diff
@@ -0,0 +1,1016 @@
+    field can be used in the [=response=] (server to client) to communicate the
+    <a>feature policy</a> that should be enforced by the client.</p>
+    <p>The header's value is the <a href="#json-serialization"></a> of the
+    elements of the <a>declared policy</a>:</p>
+    <pre class="abnf">
+      FeaturePolicy = 1#json-field-value
+                ; See Section 2 of [[HTTP-JFV]], and Section 2 of [[RFC7159]]
+    </pre>
+    <p>When the user agent receives a <code>Feature-Policy</code> header field,
+    it MUST <a href="#process-response-policy">process</a> and <a>enforce</a>
+    the serialized policy as described in <a href=
+    "#integration-with-html"></a>.</p>
+  </section>
+  <section>
+    <h3 id="feature-policy-meta-element">The <code>meta</code> element</h3>
+    <p>A {{Document}} may deliver a policy via one or more HTML <{meta}>
+    elements whose <{meta/http-equiv}> attributes are an
+    [=ASCII case-insensitive=] match for the string
+    "<code>Feature-Policy</code>". For example:</p>
+    <pre class="example">&lt;meta http-equiv="Feature-Policy" content='{"webrtc": [], "geolocation": ["https://example.com"]}'&gt;</pre>
+    <p>The <code>meta</code> policy is processed as defined in <a href=
+    "#process-meta-policy"></a> and modifications to the
+    <{meta/content}> attribute of a <{meta}> element after the
+    element has been parsed MUST be ignored.</p>
+    <p>To ensure that the policy can be applied before any scripts are able to
+    execute within the {{Document}}'s [=browsing context=], the following
+    restrictions apply to <code>meta</code> policy:</p>
+    <ul>
+      <li>The elements containing the <code>meta</code> policy MUST be
+      serialized completely within the first 1024 bytes of the document.</li>
+      <li>The elements containing the <code>meta</code> policy MUST appear
+        before any <{script}> or <{link}> elements.</li>
+    </ul>
+    <div class="note">
+      Since JSON serialization requires U+0022 QUOTATION MARK as a string
+      delimiter, it is recommended that the <{meta/content}> attribute
+      of the Feature Policy <{meta}> tag be delimited with U+0027 APOSTROPHE, to
+      avoid having to encode JSON delimeters as "<code>&amp;quot;</code>"
+    </div>
+    <div class="issue">
+      TODO: Ensure that inserting this element after scripts can run has no
+      effect.
+    </div>
+  </section>
+  <section>
+    <h3 id="iframe-allow-attribute">The <code>allow</code> attribute of the
+    <code>iframe</code> element</h3>
+    <pre class="idl">
+partial interface HTMLIFrameElement {
+    [CEReactions, SameObject, PutForwards=value] readonly attribute DOMTokenList allow;
+};</pre>
+    <p><{iframe}> elements have an "<code>allow</code>" attribute, which
+    contains an unordered set of unique space-separated tokens that are ASCII
+    case-insensitive. The allowed values are names of features. Unrecognized
```

Comment:
This keeps it in line with other iframe attributes, like sandbox. I didn't think it was useful to insist that these tokens are case-sensitive, while sandbox attributes on the same element are insensitive.